### DISCUSSION TITLE
怎么将友链访问失败的默认头像改成自己设置的？能自定义就好了

### DISCUSSION THREAD

**Discussion by MengNianxiaoyao** [YunYouJun/valaxy#267] [Q&A] [answered]
如题，头像加载不出来的就使用自己设置的，云游君的cdn有点慢。‘

还有就是这里的，每次都加载(看图)
![image](https://github.com/YunYouJun/valaxy/assets/103258286/55fddc44-15a8-4bdd-9507-c441533f74db)
影响了网站加载速度，希望这个也能自定义

**Comment by YunYouJun** [comment]
可以通过 css 变量自定义。
不过确实可以提取出配置。

**Comment by YunYouJun** [comment, marked as answer]
- 变为本地图片 + 可以通过 errorImg 自定义失败图片

![image](https://github.com/YunYouJun/valaxy/assets/25154432/49e8cd43-3f8b-4674-a165-e8c6983bc667)

Released in `v0.14.61`.
